pkg/vault/contracts/VaultFactory.sol

```diff
@@ -64,28 +74,67 @@ contract VaultFactory is Authentication {
      * @param targetAddress Expected Vault address. The function will revert if the given salt does not deploy the
      * Vault to the target address.
      */
-    function create(bytes32 salt, address targetAddress) external authenticate {
-        if (isDisabled) {
```

Comment:
I think I'd keep the flag. We don't want to deploy more than once.

Reply:
create2 doesn't allow the deployment of contracts again. We don't need a flag to protect double deployment